x-pack/plugins/apm/server/routes/service_map/get_service_map_backend_node_info.ts

```diff
@@ -75,32 +89,61 @@ export function getServiceMapBackendNodeInfo({
 
     const count = response.aggregations?.count.value ?? 0;
     const errorCount =
-      response.aggregations?.[EVENT_OUTCOME].buckets[0]?.doc_count ?? 0;
+      response.aggregations?.outcomes.buckets[0]?.doc_count ?? 0;
     const latencySum = response.aggregations?.latency_sum.value ?? 0;
 
     const avgErrorRate = errorCount / count;
-    const avgTransactionDuration = latencySum / count;
-    const avgRequestsPerMinute = calculateThroughput({
+    const latency = latencySum / count;
+    const throughput = calculateThroughputWithRange({
       start,
       end,
       value: count,
     });
 
     if (count === 0) {
       return {
-        avgErrorRate: null,
+        errorRate: undefined,
```

Comment:
```suggestion
        failureRate: undefined,
```

Reply:
I changed it to `failedTransactionsRate`